src/completion.ts

```diff
@@ -234,41 +235,43 @@ export async function asResolvedCompletionItem(
         item.additionalTextEdits = asAdditionalTextEdits(details.codeActions, filepath);
         item.command = asCommand(details.codeActions, item.data.file);
     }
-    if (features.completionSnippets && canCreateSnippetOfFunctionCall(item.kind, options)) {
-        const { line, offset } = item.data;
-        const position = Position.fromLocation({ line, offset });
-        const shouldCompleteFunction = await isValidFunctionCompletionContext(filepath, position, client);
-        if (shouldCompleteFunction) {
-            createSnippetOfFunctionCall(item, details);
+    if (document) {
+        if (features.completionSnippets && canCreateSnippetOfFunctionCall(item.kind, options)) {
```

Comment:
Can you combine those conditions to avoid creating extra indentation level?

Reply:
done